web/components/demo/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { SecondaryButton, PrimaryButton } from '../input/buttons';
+
+const ButtonCallback = e => console.log(`${e.currentTarget.textContent} button clicked!`);
+
+const FrontEndComponents = () => (
+  <div>
+    <p>Buttons</p>
+    <br />
+    <div>
+      <PrimaryButton text="Primary" onClick={ButtonCallback} />
+      &nbsp;
+      <SecondaryButton text="Secondary" onClick={ButtonCallback} />
+    </div>
+    <br />
+    <div>
+      <PrimaryButton text="Primary" onClick={ButtonCallback} disabled />
```

Comment:
I would like to see some specs that ensures `onClick` callback is not called when it's `disabled`, and that it's called when it's not `disabled`

Reply:
I'll look into adding a few simple tests for this PR, though I'll just point out that the disabled-ness is pretty enforced in the CSS:

```sass
  &:disabled
    pointer-events: none
```

Reply:
Yeah I get that's enforced in the CSS but we can't predict how the CSS file would change later on - tests help us ensure that our components are still working as expected no matter how much our code changes in the future.

Reply:
See 941495c